ocs_ci/utility/utils.py

```diff
@@ -411,13 +411,20 @@ def run_cmd(cmd, secrets=None, timeout=600, ignore_error=False, **kwargs):
         timeout=timeout,
         **kwargs
     )
-    log.debug(f"Command output: {r.stdout.decode()}")
-    if r.stderr and not r.returncode:
-        log.warning(f"Command warning: {mask_secrets(r.stderr.decode(), secrets)}")
+    if len(r.stdout) > 0:
+        log.debug(f"Command stdout: {r.stdout.decode()}")
```

Comment:
We should mask secrets for the stdout here as we do on other places to not expose the password/secret data to logs.

Reply:
Interesting, my original impression was that logging stdout via debug log level unaltered is done on purpose, I don't change this behavior in this pull request.

Reply:
I reported an issue about this https://github.com/red-hat-storage/ocs-ci/issues/1623, and will fix it here.

Reply:
Thanks, not sure why we didn't masked it before maybe was missed.
Maybe for debugging purpose it's worth to have option in our config under REPORTING section something like debug_secrets and update the function for mask secrets to check if this config value is enabled and if so it will ignore the masking secrets at all.  WDYT @mbukatov ?